### PR TITLE
Populate spreadsheet 2

### PR DESCRIPTION
**Pull request by Uriel867**
Implemented the website iterator.
each call to next() return a string array of 5 elements, each containing a single string from the website.

### CODE REVIEW

File: app/app.py [hunk 1]
```diff
@@ -75,6 +75,14 @@ def find_recipe_in_url_window_algo_based():
...
     return create_json_response(ingred_paragraph, instr_paragraph, title, url)
 
+@app.route('/populate_training_example_from_url/', methods=['POST'])
+def populate_training_example_from_url():
+    password = request.form.get('password')
+
+    if password != "toy_password":
+        return {'error': 'Invalid password'}, 401
+
+    return 'OK', 200
```
Comment (smartFreshBear) [inline app/app.py:85]: this is here because you didn't start a new feature branch.
the common practice is that each ticket require a new feature branch.
many ways to solve it.
you can do the following:
1- rename the branch to "<ticket-id>-<your-name>_<name_of_feature>" [our new convention from now on]
2- rebase on to master (read how), if there is a conflict on app.py take the changes from master.
3- squash the. commits (or just make sure there is only 1 commits)

File: training/training_population.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+from utils import text_extractor
+SIZE_OF_TRAINING_EXAMPLE = 5
+
+
+class WebsiteIterator:
+    def __init__(self, url):
```
Comment (smartFreshBear) [inline training/training_population.py:6]: this is a classic case where we need to test for logic.
this will be your "proof" you tested many edge cases.

**Review by smartFreshBear — CHANGES_REQUESTED** [review]
please attend to the comments
